src/read.rs

```diff
@@ -645,6 +645,39 @@ impl<R: Read + Seek> ZipArchive<R> {
         Ok(shared)
     }
 
+    /// Returns the verification value and salt for the AES encryption of the file
+    ///
+    /// It fails if the file is not encrypted or if the file number is invalid.
+    ///
+    /// # Returns
+    ///
+    /// - Some with the verification value and the salt
+    /// - None if the file is not encrypted with AES
+    #[cfg(feature = "aes-crypto")]
+    pub fn get_aes_verification_key_and_salt(
+        &mut self,
+        file_number: usize,
+    ) -> ZipResult<Option<(AesMode, Vec<u8>, Vec<u8>)>> {
+        let (_, data) = self
```

Comment:
As clippy says, make this a struct. And again, make the verification value an array.